### ISSUE TITLE
Progressbar with label removes last 2 lines

### ISSUE THREAD

**Issue by noplanman** [thephpleague/climate#94]
When using the progress bar with a label, the last 2 lines before the progress bar get removed/replaced/eaten up!

Should it be necessary to call `$climate->br();` twice before any progress bar with label?

Example code:

``` php
$climate = new League\CLImate\CLImate;

$climate->out('Output before progress bar.');

$climate->br();
$climate->br();

$progress = $climate->progress()->total(100);

for ($i = 0; $i <= 100; $i++) {
    $progress->current($i, "label for nr. $i");

    // Simulate something happening
    usleep(8000);
}

$climate->out('Output after progress bar.');
```

**Comment by duncan3dc** [comment]
@noplanman I can see one line being eaten but not 2, are you sure of your output lines are being eaten? I've submit the above pr to fix the one line issue

**Comment by noplanman** [comment]
@duncan3dc, you're right, it's just 1 line.

Wonder why I thought it was 2, strange 😕 

Thanks for your PR!

**Comment by duncan3dc** [comment]
This has been fixed by #98, thanks for reporting the issue @noplanman
